pages/k8s/charm-kubernetes-worker.md

```diff
@@ -352,21 +352,307 @@ update the configuration for the workers:
     ```
 
 ## Actions
-
-The kubernetes-worker charm models a few one time operations called
-[Juju actions](https://jaas.ai/docs/working-with-actions) that can be run by
-Juju users.
-
-### Pause
-
-Pausing the workload enables administrators to both drain and cordon
-a unit for maintenance.
-
-### Resume
-
-Resuming the workload will uncordon a paused unit. Workloads will automatically
-migrate unless otherwise directed via their application declaration.
-
+<!-- ACTIONS STARTS -->
+<!-- AUTOGENERATED TEXT - DO NOT EDIT -->
+
+The following actions are available. You can run an action with the following
+
+The following actions are available. You can run an action with the following ```bash juju run-action kubernetes-worker ACTION [parameters] [--wait] ```
```

Comment:
This should be multi-line like for the master

~~~suggestion
The following actions are available. You can run an action with the following:

```bash
juju run-action kubernetes-worker ACTION [parameters] [--wait] 
```
~~~